src/commands/importExportEntities/entities/previewUrls.ts

```diff
@@ -3,7 +3,10 @@ import { PreviewContracts } from "@kontent-ai/management-sdk";
 import { notNull } from "../../../utils/typeguards.js";
 import { EntityDefinition } from "../entityDefinition.js";
 
-export const previewUrlsEntity: EntityDefinition<PreviewContracts.IPreviewConfigurationContract> = {
+export const previewUrlsEntity: Omit<
```

Comment:
it's secondary, since preview urls will have been nuked already by clean of spaces and types, but since the tool supports selecting entities for cleanup, I added it.